lsp/nls/Cargo.toml

```diff
@@ -44,6 +44,9 @@ derive_more.workspace = true
 lazy_static.workspace = true
 csv.workspace = true
 thiserror.workspace = true
+crossbeam = "0.8.4"
+ipc-channel = "0.18.0"
+bincode = "1.3.3"
```

Comment:
Any reason to not use the workspace version, as elsewhere, to avoid feature unification causing breakages and to ensure we use a common version if we ever need crossbeam in another crate of the workspace?